communications/newsletters/newsletter_30_Dec2021.md

```diff
@@ -0,0 +1,253 @@
+We have allocated 30 minutes after the session (18:00 - 18:30 UTC) to have open discussions with attendees and hear their reflections on the topic.
+[Register to join the conversation](https://www.eventbrite.co.uk/e/fireside-chat-shared-concerns-in-open-research-communities-tickets-217634158627?keep_tld=1).
+
+The session will feature Emmy Tsang, Kate Hertweck, Tiago Lubiana, Humberto Julio Debat and Melissa Mendonça.
+They will bring their combined perspectives as community builders, educators, trainers, mentors, researchers, industry professionals and facilitators of open and reproducible practises at both local and global scales.
+
+We invite you to host and lead future fireside chats in your network.
+[Connect with us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch) to discuss more.
+
+![](images/2021-12-firesidechat-2.jpg)
+
+### Collaboration Cafes & Coworking Calls
+
+The last Collaboration Cafe of the year is taking place on **15 December 2021 from 15:00 - 17:00 UTC**.
+Join our project members and core contributors to introduce yourself, about your interests in research and how we can work together.
+If you haven’t yet managed to connect with *The Turing Way* community, this would be a good opportunity.
+We will return next year with another season of Collaboration Cafes on **19 January 2022**.
+Find more details about the event and sign up: [hackmd.io/@turingway/collaboration-cafe](https://hackmd.io/@turingway/collaboration-cafe).
+
+The last Coworking Call of this year is on 13 December from 11:00 - 12:00 UTC.
+Find details: [hackmd.io/@turingway/coworking-call](https://hackmd.io/@turingway/coworking-call).
+Next year, we will resume hosting them weekly on Mondays starting 10 January 2022.
+
+### Take Visible Roles in the Community
+
+![](images/2021-12-pathway-3.jpg)
+
+*The Turing Way Community, & Scriberia. (2021). Illustrations from the Turing Way book dashes. Zenodo. https://doi.org/10.5281/zenodo.3332807*
+
+As we finish three years of *The Turing Way*, we are reflecting on our growth, as a project, a community and breadth of research interests people represent.
+We have started working on our next [governance model](https://github.com/alan-turing-institute/the-turing-way/pull/2036) to ensure that the voices and ideas we represent as a community are presented in the impactful roles and decision-making processes as well.
+
+Many of our community members deliver talks and workshops regularly ([see details](https://zenodo.org/communities/the-turing-way/)), but representation within the community spaces is still limited to a few members.
+We want to learn how we can support more people in such visible roles.
+This may include joining Book Dash Committee, hosting Collaboration Cafes or Coworking Calls, leading fireside chats, documenting case studies that reflect their domains, delivering 'just in time' workshops, offering mentorship and more.
```

Comment:
```suggestion
This may include joining the Book Dash Committee, hosting Collaboration Cafes or Coworking Calls, leading fireside chats, documenting case studies that reflect their domains, delivering 'just in time' workshops, offering mentorship and more.
```